packages/synthetix-cli/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
I think we can name this package _just_ cli, so that the fully qualified package name is @synthetixio/cli

Reply:
Also, this git ignore might be redundant (see deployer package's .gitignore). The workspace ignore might be enough.

Reply:
fixed